### ISSUE TITLE
Tautuli notification sending but shinkro shows error and mal not updating

### ISSUE THREAD

**Issue by rvcabc** [varoOP/shinkro#55]
I've setup shinkro to work with tautuli by following the steps at [docs.shinkro.com](https://docs.shinkro.com), as far as I can tell, shinkro is connected to mal, and tautuli is connected to shinkro, but when I send a notification manual to test the updating of MAL, I get 

```
2024-11-25 10:28:48 TRC module=server sourceType=tautulli
2024-11-25 10:28:48 DBG Request processed duration=1.591556 module=server status=0
zerolog: could not write event: cannot decode event: invalid character ',' looking for beginning of value
```

Did I miss something, or is this normal?

**Comment by varoOP** [comment]
Can you share the payload you are sending to test ? Its probably a formatting issue

**Comment by rvcabc** [comment]
> Can you share the payload you are sending to test ? Its probably a formatting issue

Payload? Where do I find that?

**Comment by rvcabc** [comment]
Are you asking for this?

```
{"Account": {"title": "<redacted>"}, "event": "media.scrobble", "Metadata": {"title": "An Archdemon's Dilemma: How To Love Your Elf Bride - The Angelic Knight Situation Sounds Like a Mess I'd Rather Not Get Involved In", "type": "episode", "parentIndex": "1", "index": "9", "guid": "plex://episode/<redacted>", "Guid": [{"id": "imdb://tt31940008"}, {"id": "tmdb://238848"}, {"id": "tvdb://426488"}], "grandparentKey": "/library/metadata/1587", "grandparentTitle": "An Archdemon's Dilemma: How To Love Your Elf Bride", "librarySectionTitle": "TV Shows"}}
```

**Comment by rvcabc** [comment]
Ok, After looking through logs for tautuli, Plex, and Shinkro's docker-compose, I found my issue. Shinkro was trying to connect to plex on the server it was installed on (i.e. 10.10.10.15) however, Plex is located on 10.10.10.19. I edited the config.toml to reflect the correct ip for plex and now everthing is working. Honestly probably wouldnt have figured it out if you didn;t ask for payload. I was scrubbing logs looking for that word and noticed this, so Thanks. Truly appreciate the help. :rofl:
